Plugins/Detray/include/Acts/Plugins/Detray/DetrayConverter.hpp

```diff
@@ -0,0 +1,146 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Definitions/Algebra.hpp"
+#include "Acts/Geometry/GeometryContext.hpp"
+#include "Acts/Geometry/VolumeBounds.hpp"
+
+#include <vector>
+
+#include "detray/core/detector.hpp"
+#include "detray/io/frontend/payloads.hpp"
+
+// #include "detray/builders/detector_builder.hpp"
+// #include "detray/io/common/geometry_reader.hpp"
+// #include "detray/io/common/geometry_writer.hpp"
+// #include "detray/io/frontend/detector_reader.hpp"
+// #include "detray/io/frontend/detector_reader_config.hpp"
+// #include "detray/io/frontend/detector_writer.hpp"
+// #include "detray/io/frontend/implementation/json_readers.hpp"
+// #include "detray/navigation/volume_graph.hpp"
+// #include "detray/utils/consistency_checker.hpp"
```

Comment:
From what I can tell, only the ```detector_builder```, ```geometry_reader``` and ```consistency_checker``` need to be included from this block, since you are using them in this header directly. Can the rest be removed?